src/app/components/Caption/index.tsx

```diff
@@ -58,8 +58,10 @@ const renderCaption = (
   paragraphBlocks: OptimoBlock[],
   offscreenText: string,
   dir: string,
+  className?: string,
 ) => (
   <Text
+    className={className}
```

Comment:
Best practice here I think would be to convert this to a compound component, but I just needed to adjust one line of CSS, so I went for a class name override here, due to the amount of work needed to convert it